### PR TITLE
Add OADP Dashboard

### PR DESCRIPTION
**Pull request by dislbenn**
Recreating PR: https://github.com/stolostron/multicluster-observability-operator/pull/1103. The PR needed to be signed; therefore, I am recreating it.

Signed-off-by: dislbenn <dbennett@redhat.com>

### CODE REVIEW

File: examples/dashboards/oadp-dashboard/oadp-dash.yaml [hunk 1]
```diff
@@ -0,0 +1,1819 @@
...
+      "time": {
+        "from": "now-12h",
+        "to": "now"
+      },
+      "timepicker": {
+        "hidden": false
+      },
+      "timezone": "browser",
+      "title": "oadp-dashboard",
+      "uid": "b450c73156fe08fb9cbdd2bdbc1b037a6e95a27b",
```
Comment (subbarao-meduri) [inline examples/dashboards/oadp-dashboard/oadp-dash.yaml:1817]: Is the `uid` uniquely generated @dislbenn ?

File: examples/dashboards/oadp-dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- To install this dashboard on Grafana
+
+```shell
+oc create -f oadp-dash.yaml
+```
+
+&nbsp;
+
+- Mai metrics used in this dashboard
```
[the remark this answers is not in the document]
Comment (dislbenn) [reply examples/dashboards/oadp-dashboard/README.md:11]: Yes, this is meant to be `Main`, let me update that.

File: examples/dashboards/oadp-dashboard/oadp-dash.yaml (hunk 1 above)
Comment (dislbenn) [reply examples/dashboards/oadp-dashboard/oadp-dash.yaml]: I'm not sure; however, I just generated a unique `uid`, so that should fix the problem.

**Review by subbarao-meduri — APPROVED** [review]
/lgtm

**Review by subbarao-meduri — APPROVED** [review]
/lgtm

**Pull request merged**